lib/config_generator.py

```diff
@@ -0,0 +1,105 @@
+import re
```

Comment:
is this python file tabs instead of spaces? plz fix

Reply:
fixed